arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift

```diff
@@ -83,17 +54,54 @@ class TraceUtilityNetworkViewController: UIViewController, AGSGeoViewTouchDelega
         return overlay
     }()
     
-    // MARK: Initialize map and Utility Network
+    // MARK: Initialize map, utility network, and service geodatabase
     required init?(coder aDecoder: NSCoder) {
         // Create the map
         map = AGSMap(basemapStyle: .arcGISStreetsNight)
+        
         // Create the utility network, referencing the map.
         utilityNetwork = AGSUtilityNetwork(url: featureServiceURL, map: map)
-        
+        // Create the service geodatabase that matches the utility network.
+        serviceGeodatabase = AGSServiceGeodatabase(url: featureServiceURL)
         super.init(coder: aDecoder)
+        // Load the service geodatabase.
+        serviceGeodatabase.load { [weak self] error in
+            guard let self = self else { return }
+            // Create electrical distribution line layer ./115 and electrical device layer ./100.
+            self.layers = {
+                return [115, 100].map {
```

Comment:
There were a suggestion by Phil not to use the path component concatenation pattern: https://github.com/Esri/arcgis-runtime-samples-ios/pull/858#discussion_r418925647